### PR TITLE
Improve SW readibility

### PR DESCRIPTION
**Pull request by beaufortfrancois**
TBR: @jyasskin

### CODE REVIEW

File: playbulb-candle/sw.js [hunk 1]
```diff
@@ -1,27 +1,40 @@
-self.addEventListener('fetch', function(event) {
-  event.respondWith(
-    caches.open('playbulb-candle').then(cache => {
-      return cache.match(event.request).then(response => {
+const addToCache = (request, networkResponse) => {
```
Comment (jyasskin) [inline playbulb-candle/sw.js:1]: For a global function, why not

``` js
function addToCache(request, networkResponse) { …
```

?

Comment (gauntface) [inline playbulb-candle/sw.js:1]: I think this comes down to personal taste.

Comment (jyasskin) [inline playbulb-candle/sw.js:1]: Kinda. `function` is clearer that we're defining a function, it takes fewer tokens, and it's more familiar to people reading the code. `const` might be clearer that we don't overwrite the function? If there's another reason to pick `const` that's fine, but I don't think we should switch to it against the three reasons to prefer `function`.

Comment (beaufortfrancois) [inline playbulb-candle/sw.js:1]: Noted!
See https://github.com/WebBluetoothCG/demos/pull/57

**Pull request merged**